module/VuFind/src/VuFind/ILS/Driver/NoILS.php

```diff
@@ -140,22 +140,17 @@ public function getStatus($id)
     {
         $useStatus = $this->config['settings']['useStatus'] ?? 'none';
         if ($useStatus == 'custom') {
-            $status = $this->translate($this->config['Status']['status']);
+            $status = $this->translate($this->config['Status']['status'] ?? '');
             return [
                 [
                     'id' => $id,
-                    'availability' => $this->config['Status']['availability'],
+                    'availability' => (bool)($this->config['Status']['availability'] ?? false),
```

Comment:
Should you avoid casting this to bool in order to support the integer constants (and for consistency with getHolding below)?

Reply:
Oops, of course. Fixed.